applications/rag/metadata.display.yaml

```diff
@@ -233,7 +241,15 @@ spec:
           description: |-
             <li> Go to the Frontend Application</li>
             <li>If IAP is enabled, log in with your organization's credentials. SSL or cert errors indicate the cert is provisioning which takes up to 20 minutes.</li>
-            <li>If IAP is disabled, scroll to <i>Ports</i> section and initiate <b>PORT FORWARDING</b> (Run in Cloud Shell) to the front end application. Launch Fronted Chat app via <b>OPEN IN WEB PREVIEW</b> button. </li>
+            <li>If IAP is disabled:
+              <ul>
+                <li>(1) Setup gcloud in your environment</li>
+                <li>(2) Get these values from out Outputs section above: the <i>GKE Cluster Name</i>, <i>GKE Cluster Location</i>, <i>Kubernetes Namespace</i></li>
+                <li>(3) Get cluster credentials: <b>gcloud container clusters get-credentials <Gke Cluster Name> --location=<Gke Cluster Location></b> </li>
+                <li>(4) Port forward to the chat interface: <b>kubectl port-forward -n <Kubernetes Namespace> rag-frontend 8080:8080</b> </li>
```

Comment:
feel free to ignore this comment, but the `kubectl` command will look nicer if you put the namespace argument right after `kubectl`:

```
kubectl -n <Kubernetes Namespace> port-forward rag-frontend 8080:8080
```

Reply:
also should be `service/rag-frontend`